bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -328,6 +347,10 @@ spec:
                         description: Namespace determines the namespace where the
                           operator should be deployed. Defaults to rhdh-operator
                         type: string
+                      source:
```

Comment:
It does not make sense to have both source and sourceName. I think it should be removed here and in config/crd/bases/rhdh.redhat.com_orchestrators.yaml. Also in the values.yaml file, "source" should be "sourceName" for the rhdh operator subscription.

Reply:
Agreed. I will follow convention and ensure that we only use `sourceName` as it is being used elsewhere. @masayag FYI

Reply:
Agreed. Will update to use `sourceName`

Reply:
I'd prefer preserving only the `source` attribute since it matches the name of the field from the subscription API https://docs.openshift.com/container-platform/4.16/rest_api/operatorhub_apis/subscription-operators-coreos-com-v1alpha1.html#spec
and that makes it easier to be matched between the template file to the values/crd file.

Reply:
Tricky thing is that I can't change the schema in this PR. All I can do is propagate the changes introduced by running `make bundle` so that these artifacts don't show up in other PRs currently pending review and merge.

Any changes to the schema should be carried out once @jenniferubah 's PR to update to v1alpha2 is merged. Once that PR is merged I can start introducing changes since it's a new API version and we haven't released it yet.

What I can do here is to delete the `values-rc.yaml` and any other RC related value file since we know they aren't to be used.

Reply:
Sounds good.